openghg/analyse/_scenario.py

```diff
@@ -748,7 +748,10 @@ def _align_obs_footprint(
 
         if resample_to is None:
             if align_to_obs:
-                footprint_data = footprint_data.reindex_like(obs_data, method="ffill")
+                if platform == "satellite":
+                    footprint_data = footprint_data.reindex_like(obs_data, tolerance=pd.Timedelta("1ms"))
```

Comment:
Can we check if we need a `method` input for this as well? And if so can be make this `method="nearest"`?